### ISSUE TITLE
Translations wanted

### ISSUE THREAD

**Issue by domyen** [hichroma/learnstorybook.com#3]
Storybook should be approachable to all people! We need help translating this tutorial into other languages. We’re especially open to 🇨🇳 Chinese and  🇲🇽 Spanish translations.

Comment below!

**Comment by icarlossz** [comment]
Maybe I can help with the Spanish translation.

**Comment by luchux** [comment]
can help @icarlossz with Spanish if needed.

**Comment by chenka** [comment]
Hello, I would be great if you use [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) to url. e.g. `/content/ja` , `/content/es`, it looks more standard and short url

**Comment by ng-hai** [comment]
Ready for Vietnamese 🤠

**Comment by domyen** [comment]
@ng-hai definitely open to it if you're volunteering 😉

**Comment by chinanf-boy** [comment]
About the plan of translate chinese #42

**Comment by fabioscsilva** [comment]
Hi @domyen,
I can help you the the Portuguese translation

**Comment by domyen** [comment]
@fabioscsilva we'd be 💯 thrilled to have you do that! 
Please send a PR 🙏

**Comment by geromegrignon** [comment]
Hey @domyen 
I can work on French translation starting with Angular.

**Comment by domyen** [comment]
@geromegrignon Yes please! The Angular editions don't have the recently added [addon chapter](https://www.learnstorybook.com/react/en/addons/) yet, can you include that in your translation? 🙏

**Comment by geromegrignon** [comment]
ok, do you want me to make a PR to add this chapter to the Angular english version as well?

**Comment by domyen** [comment]
Yea that’d be awesome 👏

**Comment by jonniebigodes** [comment]
If you want i can help out with the portuguese translation. I will start with the react version and then move on systematically to the angular and vue documentation.

**Comment by domyen** [comment]
Yes we’d really appreciate it @jonniebigodes! @tmeasday is looking to
update the tutorial to +4.x in the next couple of days so perfect timing 👌

**Comment by tmeasday** [comment]
Just to be clear I think the written text + code samples work more or less with 4.0 (let me know if not!). It’s just the code repository is a little out of date.

**Comment by jonniebigodes** [comment]
OK, and as soon as the tutorial is done, i'll pick up on it and proceed from there and start the translation. Sounds good?

**Comment by jonniebigodes** [comment]
@tmeasday sorry it was my mistake, i understood it wrong. I'm cloning the site's repository and start working on the translations as we speak.

**Comment by moonformeli** [comment]
@tmeasday I think I can translate this into Korean, starting from React first, if you think it's okay.

**Comment by domyen** [comment]
🙏 @moonformeli I'd be thrilled about a Korean translation. 

🎉 Shoutout to @jonniebigodes for shipping Portuguese!

**Comment by icarlossz** [comment]
I can do the Vue translation to spanish 👨🏻‍💻

**Comment by domyen** [comment]
@icarlossz yes please!

**Comment by jonniebigodes** [comment]
almost finished with the vue translation

**Comment by moonformeli** [comment]
@domyen  Quick question. Is it better to do PR with fully translated files than with partially translated files? :)

**Comment by domyen** [comment]
Fully translated is a bit easier. But if you end up submitting a PR with
partial translations just mention that in the post and I’ll add a “do not
merge” tag.

**Comment by domyen** [comment]
@tmeasday just updated the English guide to the new [Storybook 5.0](https://medium.com/storybookjs/storybook-5-0-db1d0f9c83b8). 
The translations could use a review so they stay up to date! 

Mind taking a look @icarlossz @alterx @luchux @chinanf-boy @danielduan @jonniebigodes 🙏

**Comment by jonniebigodes** [comment]
@domyen i've been keeping an eye on the twitter feed and saw the update. I was looking for a update on the guide so that i could make the changes on the translations. Going to start making the necessary adjustments starting tomorrow.

**Comment by tmeasday** [comment]
I'm not sure I made any text changes -- I updated the code snippets where required (very limited) in all the translations, and all the images are shared so should have just worked. You can check commits to be totally sure of course.

However I only did the React version. If someone has the time to walk through the Vue and/or Angular tutorials and see if there any problems that would be greatly appreciated.

I updated the source code for React too to use SB5. Ideally we'd do that for the other view layers too. I figure most people are on React so that's where I've put the time for now 😓

**Comment by alterx** [comment]
@domyen, Carlos reporting for duty :P 
I'll take a look to the Spanish and English versions of Angular during this week. I think there's also a new chapter that's missing for the Angular version, I'll add that one too.

**Comment by b0g3r** [comment]
I started translating of vue part to Russian in https://github.com/b0g3r/learnstorybook.com/tree/ru

**Comment by tmeasday** [comment]
Awesome!

Tom

**Comment by Ericnunez** [comment]
@domyen Hi, I wanted to help with the Spanish translation for Vue. I didn't see anyone assigned but I wanted to make sure that I was able to contribute. If there is someone assigned can you please tell me what needs to be translated. Thanks a lot!

**Comment by icarlossz** [comment]
Hey @Ericnunez, I'm very busy rn then if you can help with the translation to spanish I can help with reviews and any help that you need!

**Comment by Ericnunez** [comment]
Hi @icarlossz , That sounds good! I''l start working on the documents and follow up when i'm ready for a review.

**Comment by Ericnunez** [comment]
@icarlossz, Hey mate, just opened a pull request for my first translation. Could you review the file when you get a chance? Thanks!

**Comment by icarlossz** [comment]
@Ericnunez Looking now!

**Comment by chinanf-boy** [comment]
@domyen Sorry, miss 
i will take a look with chinese

**Comment by chinanf-boy** [comment]
@domyen up to date #87

**Comment by cheironz** [comment]
I can do the Vue translation to Simplified Chinese.

**Comment by jhta** [comment]
I can help with Vue to Spanish

**Comment by domyen** [comment]
@jhta that'd be awesome!

**Comment by 53able** [comment]
I can make Vue translation to Japanese 🇯🇵

**Comment by domyen** [comment]
@53able we'd love your help on a Japanese version 🙏.

**Comment by queuestack** [comment]
@domyen I can help with the Korean translation for Angular!

**Comment by domyen** [comment]
That would be awesome 👏

**Comment by stephanbm** [comment]
@domyen I’m not sure if anyone has mentioned this or not, but if not I would be more than willing to translate Vue/Angular into Portuguese. I’m fluent in Brazilian Portuguese.

**Comment by domyen** [comment]
@stephanbm That'd be sweet! 

The "Intro to Storybook" guide is already translated to Portuguese but "Design Systems for Developers" is not. Would you be interested in helping with that?

@jonniebigodes expressed interest in the past so he might be keen to support you!

**Comment by jonniebigodes** [comment]
@domyen  i'm almost done with the portuguese translation for the guide "Design Systems for Developers", as i type this. It still needs to be proof read. But technically is done. What i propose and if @stephanbm doesn't mind is the following, you make a portuguese brazilian version of the guides, start with the intro to storybook and i can help you with the react part of it, should you need help and if you're willing go through and translate the "Design Systems for Developers" aswell. As probably @domyen is not aware both languages are quite similar, more even now that the [Portuguese Orthographic Agreement](https://en.wikipedia.org/wiki/Portuguese_Language_Orthographic_Agreement_of_1990) is in effect, but there are still some nuances to both variations of the language. Going over the intro guide there's some things that a portuguese would read and translate one way, that the brazilian would read and translate differently. More even with "Design Systems for Developers" guide, based on the way it's written, for instance some coloquialism that were used which i tried to translated them in a way that could be understood by both audiences. 

Based on that probably use the same approach that is used for zh-CN and zh-Tw translations that is  already in place.
Create a folder for pt-pt and another for pt-br, update the file [languageNameMap file](https://github.com/chromaui/learnstorybook.com/blob/master/src/lib/getLanguageName.js), changing it to the following:
```js
const languageNameMap = {
  en: 'English',
  es: 'Español',
  'zh-CN': '简体中文',
  'zh-TW': '繁體中文',
  'pt-pt': 'Português Portugal',
  'pt-br:' 'Português Brasil',
};
```

With this change i believe that the documentation will become more custom tailored and reach a bigger audience. 

Feel free to provide feedback and i'll make the changes accordingly

**Comment by so99ynoodles** [comment]
@domyen I can help with React translation to Japanese and maybe Korean.

**Comment by domyen** [comment]
@so99ynoodles thank you! Thrilled about the Japanese translation – it was on my wishlist from day one!

**Comment by Robin-Hoodie** [comment]
I can help with the Dutch translation

**Comment by domyen** [comment]
A Dutch translation would be incredible! 🙏

**Comment by Robin-Hoodie** [comment]
What's your view on skipping translations for technical terms for the Dutch translation? 

E.g. "story file" translates to "verhaalbestand" in Dutch, but to be honest if I were to talk to a Dutch speaking colleague I wouldn't say "verhaalbestand" as that's a "forced" translation of something we both understand perfectly well in English. 

If anything, it'd raise some eyebrows as it sounds a bit silly to translate a lot of technical terms into Dutch and we usually just use the English term when talking about technical stuff.

I'm from Belgium, but I'm pretty sure it's the same for programmers living in the Nederlands

**Comment by luchux** [comment]
Hey Robin! we have not forced translations in spanish either. Those terms
that are common names to the "programming or computers" domain, that in
general we don´t use with colleagues remains better in english!

Hear also Dominic opinion of course, but back in the days when we did
spanish that was the rule.

cheers

**Comment by domyen** [comment]
@Robin-Hoodie as @luchux mentioned, using "story" and other programing-specific nomenclature in English is the right call. That way folks can seamlessly transition from our guides to Storybook's English API docs. Thanks for asking though!

**Comment by mluisamc** [comment]
Can I help with the Spanish translation of Vue?

**Comment by domyen** [comment]
@mluisamc that'd be awesome 🙏

**Comment by renet** [comment]
I'm working on a German translation. Started with React, just yesterday. :)

**Comment by domyen** [comment]
Awesome @renet! You're a legend 👏

**Comment by renet** [comment]
Just a quick question: How about comments in the example code snippets? Translate them, or leave them as is?

**Comment by jonniebigodes** [comment]
@renet for now leave them as is. I had a recent discussion with one of the maintainers of this repo about this issue and despite being fully aware and understand that applying such translations would allow a more tailored read on what is happening. But it would lead to some extra issues further down the line.

**Comment by ozanmanav** [comment]
If you need Turkish, I can support you. :)

**Comment by jonniebigodes** [comment]
@ozanmanav that would be fantastic! I think that the team would love to have a Turkish translation added, allowing the documentation to reach even further. Can't wait to see it up here.

**Comment by domyen** [comment]
Thanks Ozan!

**Comment by renet** [comment]
Just a little fix for the first post of this issue:

**How to get started, 5.** should be updated like this:

> Run the project on localhost using `yarn dev` to confirm that everything looks OK. For now, you can preview changes by going to a url that looks like this `http://localhost:8000/intro-to-storybook/react/jp/composite-component/`

**Comment by IvanSotelo** [comment]
@domyen Can I help with the Spanish translation of Svelte and React Native?

**Comment by domyen** [comment]
@IvanSotelo that'd be really awesome for the Spanish-speaking dev community. 🎉

**Comment by nikolalardev** [comment]
Can I help with Bulgarian translation?

**Comment by domyen** [comment]
@nikolalardev yes please! I'm sure the Bulgarian frontend community would benefit from your contribution 👏

**Comment by Lutece** [comment]
@domyen HI :) 

I'd like to ask if I can start translating Korean for this project.

I asked the person who said he'd start work first, and he said he couldn't start because of personal reasons. (nickname: moonformeli)

Can I take over and start it?
and is there any guide which i can refer to translation progress?

**Comment by domyen** [comment]
@Lutece Yes please, a Korean translation sounds awesome. 🙇‍♂️

**Comment by XiKun-TW** [comment]
@domyen 
I can help with translate `design-systems-for-developers` into `Simplified Chinese`.

Started translated at: 
https://github.com/XiKun-TW/learnstorybook.com/tree/translate-design-system-for-developers-zh-CN

**Comment by ghengeveld** [comment]
That's awesome @XiKun-TW! We'd really love that.

I think we have to find someone to review it, because none of us know the language and I don't trust Google Translate to handle the subtleties.

**Comment by XiKun-TW** [comment]
> That's awesome @XiKun-TW! We'd really love that.
> 
> I think we have to find someone to review it, because none of us know the language and I don't trust Google Translate to handle the subtleties.

Yes, sure! Totally agree with `Google translate can't be trusted`.

I'll push my changes after finished all chapter. Feel free to have someone review it. lol.

**Comment by XiKun-TW** [comment]
Hi @domyen  @ghengeveld 

I already finished my translation and self review work.
Here is my pull request: https://github.com/chromaui/learnstorybook.com/pull/301

Feel free get someone to review it.

**Comment by domyen** [comment]
@XiKun-TW Amazing work, this is going to impact a lot of Chinese-speaking devs. 💪 

I will track down some proofreaders.

**Comment by andreeamaco** [comment]
I'd like to help translating to Romanian! :)

**Comment by jonniebigodes** [comment]
@andreeamaco that is awesome. Can't wait for the pr to be up. If you don't mind the translation will need to be proof read by someone,  as me personally don't speak Romanian and probably none other elements of the team. And Google translate helps out, but only to some degree. Sounds good?

**Comment by andreeamaco** [comment]
@jonniebigodes Yup that's good, I'll ask my FE colleague to collaborate and review! :)

**Comment by AnicetR** [comment]
Hello @domyen, I can help with translating the 'Intro to storybook' in French beginning with React.

**Comment by domyen** [comment]
@AnicetR A French translation would be very much appreciated 🙏

**Comment by Marie-Carrere** [comment]
Hi @domyen 
I would also like to contribute to the translation of the React part in French 🇫🇷
How can I help ? 😀

**Comment by jonniebigodes** [comment]
@AnicetR and @Marie-Carrere if you both could coordinate your efforts it would be awesome so that the translation gets merged as soon as possible. If you both don't have a means to reach out one to another. Don't forget that Storybook has a Discord server available in [here](https://discord.gg/UUt2PJb) that you both can use.

**Comment by AnicetR** [comment]
@jonniebigodes : Of course, thanks!

@Marie-Carrere : I'm gonna put the little I did on my GitHub page tonight (I can't rn). 
I'll send you an invite to the fork.

**Comment by jeanphibaconnais** [comment]
Hi @domyen, i can help to contribute vue part in French 🇫🇷 😄

**Comment by jonniebigodes** [comment]
@jeanphibaconnais me personally and the rest of the team would appreciate alot a french translation. Two things if you don't mind:

1- This will neeed to be further proofread, we'll ask around in the community for someone to proofread it as me personally my french is quite rusty and using google translator only gets us so far and probably some would not be properly translated.
2- If you can put the translation in the form of a pr with all the chapters with would be better for reviewing and merging.

Are you ok with this?

Stay safe

**Comment by jeanphibaconnais** [comment]
@jonniebigodes  Yes it's ok for me.

**Comment by domyen** [comment]
@jeanphibaconnais that'd be awesome! 🙏 +1 to @jonniebigodes comment.

**Comment by chameleonhead** [comment]
I'd like to translate react part into Japanese.

**Comment by jonniebigodes** [comment]
@chameleonhead that's fantastic. Would like to see the documentation translated into Japanese. A couple of things:
- If you can put the translation in the form of a pr with all the chapters with would be better for reviewing and merging.
- Also this will neeed to be further proofread, we'll ask around in the community for someone to proofread it as me personally my knowledge of Japanese is quite and i mean quite small, only a couple of words here and there.... and using google translator only gets us so far and probably some would not be properly translated.

Hoping for your pull request and see the documentation translated.

**Comment by SharkBaby** [comment]
I can translate Vue part to 🇨🇳 Simplified Chinese, can someone give us some guidance so that I can start my translation. thanks.

**Comment by katepark1009** [comment]
Hi! @tmeasday @domyen 
Is there any progress/updates on Korean translation?
If not, can I start Korean translation? :)

**Comment by domyen** [comment]
@katepark1009 That'd be amazing! ❤️  No updates on the translation progress from my end - you might want to check in with the contributor. FYI, we recently released a new React/English edition with the latest recommendations. So you may want to circle back on existing text to see that it matches up. 

@SharkBaby Nice! At the top of this post there's a "How to get started", which gives you an overview of the translation process. Aside from the translation, we could really use some help with updating the Vue tutorial content in English to match the latest React/English edition (we rewrote some chapters). Would that be something you're interested in?

Thanks everyone 🙇

**Comment by katepark1009** [comment]
@domyen Thank you! I don't see any diff between recent master branch and mine. I will continue working on it. (BTW, Love this tutorial👍)

**Comment by Draketheb4dass** [comment]
I can help translate to French for React.

**Comment by Draketheb4dass** [comment]
Started french translation at https://github.com/Draketheb4dass/learnstorybook.com/tree/language-french/content/intro-to-storybook/react/fr

**Comment by jonniebigodes** [comment]
@Draketheb4dass let me know when you have it all and the pull request is up so that we can proofread it and get it merged. Ping me when you have it. Sounds good?

**Comment by Draketheb4dass** [comment]
Sounds awesome man.

**Comment by sosunnyproject** [comment]
Hello @domyen ! me and three other folks are working on Design System for Storybook (https://www.learnstorybook.com/design-systems-for-developers/) translation into Korean. our repo is here: https://github.com/engko-translation/learnstorybook.com Planning to send PR within a week or so :)

**Comment by domyen** [comment]
@sosunnyproject that's an amazing contribution and very welcome. 🙏 @jonniebigodes is working on an updating the design systems guide (English edition) to 6.0 this week. That includes minor tweaks per the new Storybook APIs, we'll be sure to link you to that PR so you can update the Korean edition. 

Again, much appreciated!

**Comment by BraianMendes** [comment]
Hello! Regards from Brazil
I'm a React and React Native developer and I would be honored to help translate into Portuguese.
I can start helping with React Native, and even helping @jonniebigodes with the React LearnStorybook translation.

**Comment by domyen** [comment]
Good timing Braian! A new edition of the React English guide was released
last week that reflects Storybook 6.0 (released a few weeks ago).

Would you be interested in updating the React Portuguese version to the
latest writing/code based on the React English guide? That would be super
helpful and very much appreciated. 🙇‍♂️

**Comment by jonniebigodes** [comment]
@BraianMendes i would be glad to "join forces" (pardon the bad pun) so that we can adjust the translation so that not only the European Portuguese and Brazilian Portuguese and also the other Portuguese spread around the world can benefit from it. And @domyen touched a rather important point. If you're willing and start with the React version, would be great, as it needs a bit of attention. And then we could move to the React Native, as that one will probably need a bit more of work. 

Sounds good?

**Comment by BraianMendes** [comment]
@domyen I am very interested. Thank you very much for the welcome, and I believe that I really arrived at a good time to really contribute. I am very excited, I am having the opportunity to study the Storybook and implement it in my work, so for me it will be a great experience.
@jonniebigodes Sounds really good to me. We can work in pairs, and thereby optimize the work well and not get stuck in anything. We can actually do it first for React, which is the first contact material in the Storybook documentation, and then the React Native translation will be much more natural and quality for us.

When can we start? Best regards from Brazil

**Comment by josetucuman** [comment]
I can help translate from English to Spanish

**Comment by taechonkim** [comment]
Hey I want to translate to Korean for React with my co-worker (a.k.a eunkyeng)
I'm an essay writer (working as web frontend developer too)

**Comment by domyen** [comment]
@taechonkim, we're fortunate to have Korean translations in progress. 
- @katepark1009 is leading the translation for "Intro to Storybook" 
- @sosunnyproject expressed interest in the "Design Systems for Developers" guide

Can you coordinate with one of them on helping out? 🙏 

---

@sosunnyproject we merged the new edition of "Design Systems for Developers". The instructions are now stable. That means you're OK to begin translating. 

--- 

@josetucuman we could use help on translating "Design Systems for Developers" from English to Spanish.   Is that something you're interested in? 🙇

**Comment by taechonkim** [comment]
@domyen Ok I like it! How can I help them? Just pick one of them(Intro to Storybook or Design Systems for Developers), and keep going from How to get started, is it enough? If not, should I contact them(katepart1009 or sosunnyproject)?

**Comment by katepark1009** [comment]
Hi @taechonkim 
I've been working on `Intro to Storybook` React translations. 
The work is done and it's in PR. (https://github.com/chromaui/learnstorybook.com/pull/348)
Currently I'm updating the translation with Storybook 6 and revising it. 
It would be awesome if you can help me proofread it.😄

Stay safe!

**Comment by asyraf-labs** [comment]
Hi @domyen , I can help translate from English to Malay/Indonesia language.... It would be great as I can learn while translating your awesome work @domyen 🔥

**Comment by domyen** [comment]
Thanks Amirul, that would be super helpful for all the Malaysian and
Indonesian developers 🙏

**Comment by jonniebigodes** [comment]
@josetucuman like @domyen mentioned if you're interested in translating the design systems it would help us out immensely. But if you want to "start small" and learn more about Storybook, you could start with the React Intro to Storybook Spanish translation as that translation is a bit outdated and in need of some attention. With that when and if you are willing to translate the Design Systems for Developers tutorial, you'll see some familiar content and workflows that are used. 

Let me know what you think.

@BraianMendes if you're ok with the approach on this, i would love to help you out on the portuguese translations, my i'm going to put myself on the "helping hand" (pardon the bad pun) side of things. When you're able start the updates on the React version and ping me in the [Storybook Discord Server](https://discord.gg/UUt2PJb), so that we can consolidate the translation a priori and we get it merged faster. 

Sounds good?


All of you stay safe

**Comment by chameleonhead** [comment]
Hi, I'm goint to help translating Design Systems for Developers into Japanese. I'll send PR when I have finished.

**Comment by sosunnyproject** [comment]
@domyen thanks for the update! will merge the updated version of  "Design Systems for Developers" guide and will send PR soon! Currently in the process of proofreading the translated version with collaborators

**Comment by kevinOriginal** [comment]
@sosunnyproject Hi, I've took a look at [your repo](https://github.com/engko-translation/learnstorybook.com)
I think you did a fantastic job. By looking at your PRs, I'm seeing some missing chapters and wondering if you need any help on any of them like chapter 2,6,8. If you need any help, I'll be able to hop in right away. 


@domyen Hi, I've checked that you are still in WIP on ''Visual Testing Handbook". Would it be possible to get a preview? It'd be great if I could get hands on it. I'm really looking into snapshot & UI testing these days, and I think I could help some translation in Korean ahead of time on the last guide.

**Comment by jonniebigodes** [comment]
@kevinOriginal the Visual Testing Handbook is being worked on. We've been thinking about it and see how we could better present the information to you and the rest of the community. Once we get it in the documentation right, we'll let you know and we'll love for a Korean translation to be done. 

Thank you very much for taking the time in checking in the repo for @sosunnyproject and the Visual Testing Handbook.

Can't wait for the Design Systems to be also translated as well :)

Stay safe.

**Comment by kevinOriginal** [comment]
@jonniebigodes Okay. Looking foward to it! Is there an estimate time of completion? Or maybe at least an open preview?

**Comment by jonniebigodes** [comment]
@kevinOriginal sorry for not giving you an accurate timetable for now. This is something that is being researched and consolidated. Once we have something more concrete we'll be sure to let you know and would love a translation. 

Do you mind waiting a bit longer and continue to help out the Design Systems translation?

**Comment by STPace** [comment]
@domyen Hi, I can help translate the intro-to-storybook/react to Simplified Chinese, this part has not been updated.

**Comment by jonniebigodes** [comment]
@STPace we would love it. That translation is in dire need of some updates. If you can updated we would be extremely thankful for it.  Let me know when you have it so that we can ask in the community for someone to proofread it and then merged. 

Stay safe

**Comment by STPace** [comment]
@jonniebigodes Okay, I will submit pr in the near future

**Comment by jonniebigodes** [comment]
@STPace looking forward for it :)

**Comment by icruzr93** [comment]
Hi guys some free time on my work right now, I would be very proud to contribute with updating or adding some Spanish translations where required, just let me know. (I have already read **The Intro For React** and the **Design System for Developers** course).

**Comment by jonniebigodes** [comment]
@icruzr93 we would really appreciate it, as the React Spanish translation for Intro to Storybook is in dire need of some updates. We can start with that one. Let me know when you the pull request up and we'll go from there. Sounds good?

**Comment by icruzr93** [comment]
@jonniebigodes sounds good, I'll catch you later with some translations

**Comment by Romainpetit** [comment]
@jeanphibaconnais @jonniebigodes I'd be glad to help on a french translation of the Vue guide aswell. 
Is anything started yet, that I can contribute to ?

**Comment by jeanphibaconnais** [comment]
Hi @Romainpetit , for the first part, introduction, the traduction is ok. 
I haven't start the second and third part

**Comment by Romainpetit** [comment]
Well done. Do you have an open PR ?

**Comment by jeanphibaconnais** [comment]
Not yet @Romainpetit , you can do it if you want. i have another project in progress that I prefer finish little things
If you want some help, don't hesitate to ping me

**Comment by lala-lee-jobs** [comment]
I can do the Vue/Traditional Chinese translation if you want.

**Comment by jonniebigodes** [comment]
@lala-lee-jobs that would fantastic, please do. We would more than welcome it. Ping me when you have the pull request is up.

**Comment by lala-lee-jobs** [comment]
@jonniebigodes Okay, I will use Vue/English as the source in my translations.

**Comment by jonniebigodes** [comment]
@lala-lee-jobs thank you for picking up on this and your willingness to translate the documentation. As the docs are a ever mutating organism if any changes drop in i'll let you know and you adjust accordingly and we get the most recent content out. Sounds good?

**Comment by lala-lee-jobs** [comment]
@jonniebigodes OK, no problem!

**Comment by JoSaGuDu** [comment]
Hi, I can help with Spanish translation for Angular tutorial

**Comment by luisabellan** [comment]
Hello, there. I can help with Spanish translations.

**Comment by Turkyden** [comment]
Hi, I'm goint to help translating Design Systems for Developers into Chinese. I'll send PR when I have finished.

**Comment by domyen** [comment]
Thanks @JoSaGuDu @luisabellan!
For Spanish, the Design Systems for Developer tutorial is a good place to start. I believe we have Spanish translations for Intro to SB – @jonniebigodes would know more about how up to date they are for 6.0.

@Turkyden That sounds great!

**Comment by jonniebigodes** [comment]
@luisabellan really appreciate the interest in translating the tutorials to Spanish. As of now the Intro to Storybook Spanish translations are not updated for 6.0. If you want to start small and get more acquainted with Storybook, start with the Intro to Storybook tutorial, i'll leave the framework up to you. Then feel free to move to the Design Systems tutorial. As you'll see that the knowledge you've picked up in the Intro to Storybook will help you immensely with the tutorial. Let me know once the translation is updated.

@Turkyden looking forward for the pull request with the translation.

**Comment by luisabellan** [comment]
Hello, @jonniebigodes. I'm going to be very busy for the rest of the week but I'll definitely get started next week :+1:

**Comment by jonniebigodes** [comment]
Hey! @luisabellan , no problem whatsoever. We're thankful for the help in translating the tutorials! Take your time and let me know when you have the pull request up.

**Comment by sosunnyproject** [comment]
hi @kevinOriginal  
excuse my late notification check. i just saw your ping on this thread. Me and my team's translation is already merged, but i'm sure you're more than welcome to create commits for improvement. We did peer reviews internally, but please go ahead and review the docs. if you have any question, feel free to reach me via email - will be more responsive there. Thank you!

**Comment by Magicianred** [comment]
Ciao,
I am doing Italian language translate

https://github.com/Magicianred/learnstorybook.com/projects/1

**Comment by jonniebigodes** [comment]
@Magicianred that is fantastic to hear. Really appreciate the work you're doing.Let me know once the pull request is up so that we can get a proofreader and get it merged.

**Comment by krishnatandon1208** [comment]
Hi all,
I am willing to contribute to the language "Storybook in Hindi", one of the most popular languages in our country and widely spoken.

**Comment by jonniebigodes** [comment]
@krishnatandon1208 that is awesome, we'd love to see a Hindi translation done. Let me know if you run into any issues and when you have the pull request with the full translation up.

Stay safe

**Comment by jeanphibaconnais** [comment]
> Not yet @Romainpetit , you can do it if you want. i have another project in progress that I prefer finish little things
> If you want some help, don't hesitate to ping me

Hi  @Romainpetit , do you start something ? i didn't see some issue ou pull request. I can create one issue, i think i can start translation of migration storybook 6.0 in January.

**Comment by jonniebigodes** [comment]
@jeanphibaconnais no pull request was added by @Romainpetit so far. If you both are ok with a middle ground regarding the translation, @jeanphibaconnais if you're able to update the translation you've done to version Storybook 6.0 it would be an awesome improvement as you were the primary on that and @Romainpetit if you're able to proofread it then it would fantastic and we'd get it merged even sooner. If by any chance you're not able to migrate during this month then we could "turn the tables around" and let @Romainpetit update it and @jeanphibaconnais you could review it? 

Let me know what both you think and we'll proceed from there. Really looking forward to get this version updated and continue to close the loop and get all the translations updated.

Stay safe both of you

**Comment by jeanphibaconnais** [comment]
It's ok for me. I've just created a draft PR : https://github.com/chromaui/learnstorybook.com/pull/434

**Comment by jonniebigodes** [comment]
I just got the notification for the pull request 😀  ! And i'm good with it. Let me know when you've updated it to 6.x and we'll move from there. @Romainpetit as @jeanphibaconnais has started the translation, if you don't mind we'd love for your help as a proofreader. If you both want to coordinate your efforts better don't forget that we have a [discord server](https://discord.gg/UUt2PJb) and if you both need my help you can ping me there (same username).

**Comment by Romainpetit** [comment]
> It's ok for me. I've just init a draft PR : https://github.com/chromaui/learnstorybook.com/pull/434 

@jeanphibaconnais Well done ! I didn't publish anything so it's perfect you started it. I can proofread it as @jonniebigodes suggests 👌

**Comment by jonniebigodes** [comment]
That's great. I'll keep an eye on the progress, but as i'm spread around multiple sources of documentation i'll leave to both of you to coordinate your efforts and let me know once it's good to go and we'll get it merged.

**Comment by krishnatandon1208** [comment]
> @krishnatandon1208 that is awesome, we'd love to see a Hindi translation done. Let me know if you run into any issues and when you have the pull request with the full translation up.
> 
> Stay safe

I am facing an issue with pull request. Unable to compare with the existing branch. Is a new branch is to be created ?

**Comment by jonniebigodes** [comment]
@krishnatandon1208 did you fork the LearnStorybook repo? It could be related to it. Try the following:
- Fork the LearnStorybook repository.
- Create a branch on your fork with the translation.
- Update the documentation
- Make the pull request

Let me know if you have any issues and we'll go through them. 

Stay safe

**Comment by jeanphibaconnais** [comment]
Hi @Romainpetit , if you have some time, i've created this PR https://github.com/chromaui/learnstorybook.com/pull/434

**Comment by nagashimam** [comment]
Hello, I’d like to help with Japanese translation for Angular tutorial.

**Comment by jonniebigodes** [comment]
@nagashimam sure. We'd love to get a Japanese translation for Angular. Let us know when you have the pull request up and we'll get a proofreader from the community to help us with the process and get it merged.

**Comment by nagashimam** [comment]
Hi @domyen, let me clarify one thing. 
 
Your explanation at the top of this page says
> Note: React/English is the "master" edition that is updated often. Please use React/English as the source-of-truth in your translations. 

This means translation of Angular tutorial to Japanese (or any other language) should be based on React/English version except things that are specific to Angular (such as Storybook installation), right?

**Comment by jonniebigodes** [comment]
Hi there @nagashimam, regarding your comment above, yes the React version is the source of truth and each other version of the tutorial is based off that one and adjusted accordingly, such as  you mentioned, the Storybook installation and how components/stories are implemented for the framework and how you handle data. 

Also wanted to give you a bit of a heads up that the Angular version is going to be updated during this week. I'll let you know once the pull request is merged so that you can adjust accordingly. Sounds good?

**Comment by nagashimam** [comment]
@jonniebigodes 
Thank you so much, sounds great! I can't wait to read the new version.

**Comment by micksp** [comment]
I can do vue translation in Dutch

**Comment by jonniebigodes** [comment]
@micksp that sounds awesome. We'd love a Dutch translation for the tutorials. Let me know when it's up or if you have any questions and we'll go from there.

**Comment by micksp** [comment]
I'm halfway through. During the translation process I am creating a list of words that I think should not be translated from English, because they are very common among developers and would feel artificial if translated. There's also translation decisions on my list. I think there should be a list per language so these decisions stay the same for all pages in a given language, no matter who translated them. Currently I store my list-of-exceptions-and-decisions in the directory where I do my translating, but obviously that's not a good place for that.

**Comment by nagashimam** [comment]
@jonniebigodes @chameleonhead
What do you think about @micksp 's idea? I also had several words and phrases I wasn't quit sure if I should translate into Japanese or not. I just followed React/ja translation by chameleonhead, but it's nice to have such list to make sure we're consistent.

(and just so you know, I think I can get draft of Angular/ja translation ready this week. Translation is done and I want to proofread by myself this weekend.)

**Comment by micksp** [comment]
I also notice a cultural difference that I'm trying to incorporate. The American (I guess) style using lot's of "fantastic" "great" and "let's do this" - this is not the way European people talk. So I find I am downsizing the superlative tone to make it more compatibel with the way Dutch people talk, instead of literally translating the words.

**Comment by eihabkhan** [comment]
Hey @domyen 👋
I would love to help with an Arabic translation for React

**Comment by domyen** [comment]
@eihabkhan Awesome, that's going to be really helpful for our Arabic readers! 🙇‍♂️

**Comment by chameleonhead** [comment]
@nagashimam Sorry for slow response. I agree to match words in translation. If you make English-Japanese corpus, I'll adjust translation with that.

@domyen @jonniebigodes Where can we put per-laguage corpus or guidelines? Or just create issue?

**Comment by delfilb** [comment]
Hi @icarlossz  @luchux! I would like to contribute with the Spanish translation.

**Comment by jonniebigodes** [comment]
@delfilb that's great to hear. We'd love to get the current Spanish translations updated and new ones added. f you're new to Storybook and want to start small. We just released a small handbook called [Visual Testing Handbook ](https://storybook.js.org/tutorials/visual-testing-handbook/) and we'd love a Spanish translation. 

Let me know and we'll go from there.

**Comment by K-Schaeffer** [comment]
Hello, I'd like to contribute with the Portuguese (BR) translation, I already started translating the Vistual Testing Handbook, but didn't open a PR or issue yet.

**Comment by delfilb** [comment]
Hi @jonniebigodes. That sounds great! I'm new to Storybook, this will be my first contribution so any advice is welcome. If it's ok I will start with the translation of the Virtual Testing Handbook right away.

**Comment by jonniebigodes** [comment]
@delfilb sure, please do we'll love a translation for the tutorial. Let me know if you run into any issues and we'll go from there.

@K-Schaeffer we'd love a translation as well. One thing regarding this particular item. If you're ok instead of making a pt-br /pt-pt translation we can probably create a unified translation for all native Portuguese speakers/writers? The rationale behind this is that it would be simpler to maintain and improve in the future. I'm aware of the way and nuances to both "forms of the language" (pardon the bad pun), but with this approach, we keep a well-maintained translation which we can improve with minimal effort from either you or me or any other contributor.

Let me know and we'll go from there.

Stay safe both of you!

**Comment by K-Schaeffer** [comment]
@jonniebigodes That's not a problem at all, we can surely do it, and I'm really looking forward to help with it :)

**Comment by IvanHatt** [comment]
@icarlossz  @luchux 
Hi, I would like to help with spanish translation "Tutorials: Create an Addon"
(I'm a native speaker, born in Argentina)

**Comment by jonniebigodes** [comment]
@IvanHatt you're more than welcome to translate the tutorial, we really appreciate it. Just let me know once the translation is up and we'll reach out to the community and find a proofreader to take a look at it and once it's all good I'll be more than glad to merge it.

Stay safe

**Comment by alessiaAmitrano** [comment]
Hi there, I can work on the Italian translations for both Angular and React!

**Comment by ymcheung** [comment]
Hi!
Just noticed the flag in Traditional Chinese is 🇨🇳 (China).
<img width="608" alt="CleanShot 2021-08-05 at 11 00 17@2x" src="https://user-images.githubusercontent.com/4409977/128284378-c21053de-3eb3-4c58-9445-ddb4e4a966bc.png">
However, few people use Traditional Chinese in China.
It is great to use both 🇹🇼 and 🇭🇰 for Traditional Chinese.

**Comment by jonniebigodes** [comment]
Hey @ymcheung thanks for the clarification on this, I'll adjust accordingly. 

Stay safe

**Comment by jonniebigodes** [comment]
@ymcheung all done. Once again thanks for the clarification on this. We appreciate it.

**Comment by naoki8ster** [comment]
Hello, I would like to help with Japanese translation of Design Systems for Developers tutorial.
I’m a front-end engineer having motivation towards design systems.

**Comment by jonniebigodes** [comment]
@naoki8ster thanks for the interest and willingness to help out with the Design Systems for Developers tutorial, we appreciate it. Let me know when the tutorial's translation is up so that we can get a proofreader from our community that is willing to read it and fine-tune it so that we can merge it.

Stay safe

**Comment by zaki-yama** [comment]
Hello, I'm Japanese web developer.
I recently read [UI Testing Handbook](https://storybook.js.org/tutorials/ui-testing-handbook/) and that was really great!
I'd be happy if I could help to translate it into Japanese.

**Comment by jonniebigodes** [comment]
@zaki-yama that's fantastic, glad that you liked the handbook. We'd love to get a Japanese translation included so that it can help both you and the Japanese frontend community. Looking forward to the pull request, let me know once it's up.

Have a great day!

Stay safe

**Comment by zaki-yama** [comment]
@jonniebigodes 
Thanks for replying. I'll try it!
I just wondered if there were someone who can review Japanese. Do you have any Japanese core reviewers?

**Comment by jonniebigodes** [comment]
@zaki-yama no need to thank whatsoever! The previous translations were done by @chameleonhead (check at the top of this issue for the community translators). Once the pull request I can reach out to him to see if he's available to review the tutorial, if not we'll ask in our community for someone to help us proofread it and get it merged. Sound good?

**Comment by zaki-yama** [comment]
@jonniebigodes 
Great! I understood. Thank you!

**Comment by naoki8ster** [comment]
@jonniebigodes  Hi, I've just finished translating Design Systems for Developers into Japanese so I am going to push my branched repository into remote repo, but I got the following permission error from server. Could you tell me how to make a PR?
Thanks.

```
remote: Permission to chromaui/learnstorybook.com.git denied to naoki8ster.
fatal: unable to access 'https://github.com/chromaui/learnstorybook.com.git/': The requested URL returned error: 403
```

**Comment by jonniebigodes** [comment]
Hi @naoki8ster ! Wow, that's great to hear that you translated the tutorial to Japanese, we're stoked to get it added in to the tutorials and with that help the community.

To give you a bit of context on your issue what's happening is that you're not a part of the organization (the ChromaUI organization). In order to get the pull request added to this repository, you'll need to take a few steps:
- Fork this repo (you'll see the button at the top of the page)
- Clone your fork locally
- Create a branch for your changes (e.g., git checkout -b "DSD-japanese-translation")
- Add the changes you've made, commit them, and push them into your forked repository. Once you've opened your fork in here (GitHub), you'll see the option to make a pull request, follow the instructions and the pull request will be added here.

To give you a bit of help on this, checkout this [link](https://www.freecodecamp.org/news/how-to-contribute-to-open-source-projects-beginners-guide/) for some documentation on how to contribute to open-source projects. I'm aware that's a bit extensive, but I can help answer some of the questions you have. Let me know if you have any additional questions and I'll gladly follow up with you on this.

Hope you have a great week!

Stay safe.

**Comment by naoki8ster** [comment]
@jonniebigodes  Thank you for your polite guidance of PR! I understood the reason why I couldn't push that. I've just opened [a pull request](https://github.com/chromaui/learnstorybook.com/pull/573) following above instructions. I'm glad my translation to be reviewed by a proofreader, and also thanks for giving me a helpful article so I'm going to check it out :)

**Comment by jonniebigodes** [comment]
@naoki8ster no need to thank whatsoever! It was my pleasure to help you get onboarded and be able to contribute to your translation. I'll take a look at it tomorrow from a more technical side, and ask one other Japanese native speaker that contributed to the tutorials earlier to proofread it. If he's unable to do it for some reason, I'll reach out to our community and see if there's another member that is willing to coordinate with you and me and get the translation in. Sounds good?

**Comment by naoki8ster** [comment]
@jonniebigodes Awesome! I appreciate these process, and I'm looking forward to be reviewed :)

**Comment by jeanphibaconnais** [comment]
Hi @jonniebigodes , i think [this PR](https://github.com/chromaui/learnstorybook.com/pull/434) has so many time and there are so much news that if you want, you can close it and i will re translate the french part.

**Comment by jonniebigodes** [comment]
@jeanphibaconnais sorry for the delayed response, I've been working on some other aspects of the documentation and it slipped my mind. For that, I'm truly sorry 🙏 ! I'm ok with your suggestion. I'll close it and we can start fresh. Looking forward to the updated version! 

Hope you have a great weekend!

Stay safe

**Comment by AilinAsis** [comment]
Hi! I would like to help. I can to translate to Spanish.  @domyen

**Comment by jonniebigodes** [comment]
@AilinAsis we'd love some help with the Spanish translations, as they're in need of some much-required updates! Let me know if you need any help! Looking forward to some much-needed updates! 

Hope you have a fantastic weekend!


Stay safe

**Comment by ailinasis096** [comment]
@jonniebigodes Perfect!. I'm new in Storybook, this will be my first contribution so any advice is welcome.

**Comment by Magicianred** [comment]
Hi, I have a question. I see there is some index.md page that are outside the translations folders.
These pages should be translate or not? If yes, where is the correct place for them?
thanks

I have need of help for the Italian translations. We have already started to translate, we need someone for reviews the translation and for other translations. Thanks

Ho bisogno di aiuto per la traduzione in italiano. Abbiamo già iniziato a tradurre, abbiamo bisogno di qualcuno per revisionare le traduzioni e per farne nuove. Se qualcuno è interessato trova qui il project per contribuire. grazie

https://github.com/Magicianred/learnstorybook.com/projects/1

**Comment by jonniebigodes** [comment]
@Magicianred that's great to hear that you already started the translation! To answer your question, these [index files](https://github.com/chromaui/learnstorybook.com/blob/master/content/intro-to-storybook/index.md) don't need translations at all. Don't worry. Keep focusing on the translation. Also if you're ok with it, share the work you've done with our community in our [Discord server](https://discord.gg/storybook) also if you have a Twitter account tweet about it and mention Storybook and i'll gladly retweet it for visibility. What do you think? Sounds good?

@AilinAsis a good advice to get you started would be to go through the Intro to Storybook tutorial in English, starting with the framework you feel more familiar with and get to know some of the concepts, for instance, args, decorators and so on. As getting to know those features will help you out immensely when translating the tutorials. 


Looking forward to hear from both of you!

Hope y'all have a great day!

**Comment by watsuyo** [comment]
@jonniebigodes 
HI! I'm Web Front-End Developer. I work with Vue.js.
I interested in translating [intro-to-storybook](https://github.com/chromaui/learnstorybook.com/tree/master/content/intro-to-storybook)/vue.

If I can help, I would like to contribute.

**Comment by jonniebigodes** [comment]
Hey 👋  @watsuyo! wow, that's amazing to hear and by all means please do, I'll be looking forward to it and once it's up I can reach out to our community and get someone to proofread it and help you and me get it merged. Let me know if you have any questions and I'll gladly answer them.

Hope you have a fantastic day!

Stay safe

**Comment by nabeken5** [comment]
Hi, I am a web developer in Japan.
I recently learned about Storybook and was really surprised to see how useful it is.
I would be happy if I could help you translate it into Japanese.

**Comment by opmalzahn** [comment]
Hi, I'd like to help with vue/spanish translation. It will my first time contributing to a project so I hope I can be useful :)

**Comment by jonniebigodes** [comment]
@opmalzahn that's great to hear and by all means please do. Looking forward to the updated translation. Let me know if you need any help. 

Hope you have a great weekend.

**Comment by jonniebigodes** [comment]
@iTarka wow, that's amazing to hear. If you could add either one, we'd appreciate it dearly. Looking forward to seeing it. Let us know if you need any help getting set up.

**Comment by stereodenis** [comment]
I can help with translating into Russian

**Comment by aso1datov** [comment]
> I can help with translating into Russian

Hey, I also want to translate, let's collaborate. I've created [issue](https://github.com/chromaui/learnstorybook.com/issues/623) with a plan an [PR](https://github.com/chromaui/learnstorybook.com/pull/622) with translation of "Get started page".

**Comment by jonniebigodes** [comment]
@stereodenis and @aso1datov if you're both ok with collaborating would be amazing, also thanks for taking the initiative and setting up an issue with the progress. I'll be on the lookout and check the translation as it progresses. Can't wait to see it materialized and released to the community. Thanks for taking the time and effort for this, appreciate it 🙏

**Comment by geekcatfr** [comment]
Can help with french translation!

**Comment by jeanphibaconnais** [comment]
Yes of course @jonniebigodes  😄

**Comment by jeanphibaconnais** [comment]
Hello @harryssecret , can I help you on this translation ? I was busy these previous month but I think I can be available in september to translate or help you to translate in 🇫🇷 
Are you connected in the discord server ? it would be easier to exchange that on this issue (and avoid to spam everyone 😃 )

**Comment by harryssecret** [comment]
> Hello @harryssecret , can I help you on this translation ? I was busy these previous month but I think I can be available in september to translate or help you to translate in fr Are you connected in the discord server ? it would be easier to exchange that on this issue (and avoid to spam everyone smiley )

Yes for sure ! I've started the translation but left it for a while:)) my discord is letsallmeow#9085, i'm not on the discord server for the moment !

**Comment by jeanphibaconnais** [comment]
Hello @harryssecret , I saw that you started translating on React. Do you imagine staying on this framework ? For my part I worked on Vuejs part, so I can start translation on it if you don't imagine to work on this part 😄 
(come, come to the discord server 😃 )

**Comment by shogohida** [comment]
Hi @jonniebigodes, I want to contribute to translation! I already made 2 pull requests for Spanish and Portuguese.

https://github.com/chromaui/learnstorybook.com/pull/664
https://github.com/chromaui/learnstorybook.com/pull/665

I'm Japanese so I will help with Japanese translation too!

**Comment by jonniebigodes** [comment]
@shogohida thank you for your interest in helping us out. We appreciate it 🙏 ! I'm reviewing the Portuguese translation as I type this and will take a look at the Spanish one tomorrow morning (Monday) my time. And please feel free to submit a pull request for the Japanese translations we'd welcome them.

Hope you have a great weekend.

Stay safe

**Comment by nagashimam** [comment]
Hi @jonniebigodes, Japanese translation of Angular tutorial hasn't been updated for a while. Can I update it?

**Comment by jonniebigodes** [comment]
Hey @nagashimam, please feel free to work on it and let me know once the pull request is up so I can reach out to the community for a proofreader. Sounds good?

Looking forward to seeing the updates. 

Hope you have a fantastic weekend.

Stay safe

**Comment by lmac-1** [comment]
Hello! Do you still need some help updating the Spanish translations? I've just been going through the tutorials (in English), and they really are great! 

Have a great day :)

**Comment by lmac-1** [comment]
Hey @jonniebigodes thanks for the reply. I'm originally from the UK but I'm fluent in Spanish and living in Colombia. 

What's the general process? Should PRs be raised for each Markdown file? Do you need me to find someone native to help proof read my translations? Or do you have someone already?

Where would be best to keep track of this? I have quite a few developer contacts on Twitter in Colombia and Latin America, so perhaps I could get a group going to help translate and proof read.

Have a great day, and looking forward to getting started!

**Comment by jonniebigodes** [comment]
Hey, @lmac-1 👋  thanks for following up with me on this and your spot-on questions. The general rule of thumb is to have the full translation added into a single pull request so that it can be proofread more easily and thoroughly. Regarding proofreading, we generally call out folks that contributed in the past or send out a message on our Twitter / Discord for someone to help, but if you already have a group of folks that can help you out, it would be fantastic. In the past, the Korean translations were done similarly as you're describing, with a small team of people updating the tutorials and proofreading each other's work, and it required little to less involvement from me, just some minor tweaks here and there, and it was published. If you're okay with that approach, it would be fantastic. 

Looking forward to hearing from you and and thrilled that you're willing to start the work. That's great to hear.

Have a fantastic day.

Stay safe

**Comment by lmac-1** [comment]
Great! I will start working on translating the `intro-to-storybook` tutorial. When I make a PR, what branch should I be pointing to? I noticed there's a `language-spanish` branch but it's very behind! 

Thanks, 
Lucy

**Comment by jonniebigodes** [comment]
Really?🤩 That's fantastic to hear, and you're more than ok to target the `master` branch for your pull request. The one you've mentioned is outdated and should not be used. This was a bit of a mishap from us in managing the branches currently listed in this repository. Let me know once the pull request is up, and we'll go from there.

Have a great day. Stay safe

**Comment by lmac-1** [comment]
Just a quick update - I'm still working on translating the "Intro to Storybook" tutorial on my fork. I've done first draft translations of Get started, Simple component, Composite component and Data and I'm hoping to work through the rest this week :)

**Comment by jonniebigodes** [comment]
@lmac-1, thanks for checking in and giving me a status update. That's great news, I appreciate it 🙏 ! Looking forward to seeing it.

**Comment by xaviermarchal** [comment]
Hello @jonniebigodes, do you need help for french 🇫🇷 translations? 

I'm a french native but pretty fluent in english, so i guess i would be able to translate the md files well.

Is the status list up to date? As i mostly work with React and RN, I was thinking about starting with the following files if you need help:

- [learnstorybook.com/content/intro-to-storybook/react-native/](https://github.com/chromaui/learnstorybook.com/tree/master/content/intro-to-storybook/react-native)
- [learnstorybook.com/content/design-systems-for-developers/react/](https://github.com/chromaui/learnstorybook.com/tree/master/content/design-systems-for-developers/react)

What do you think? Should i start or is it already done by someone else?

**Comment by jonniebigodes** [comment]
@xaviermarchal, thanks for your interest in helping us out with the translations, and yes, yes, we do need help as they are a bit outdated compared to the current English version. 🙏 ! If you're ok with it, I would recommend with the React version first before moving on to the React Native version, as it's currently quite outdated, and I'm working with the React Native maintainer to get it updated. If you can make the adjustments soon, before the 7.0 release, I can reach out to the community and see if anyone is interested, or if you know someone that is willing to collaborate with you in proofreading it, I'd appreciate it.

Let me know and we'll go from there.

Hoping to see the pull request with the updates to the translation.

I hope you have a great day.

Stay safe

**Comment by xaviermarchal** [comment]
> @xaviermarchal, thanks for your interest in helping us out with the translations, and yes, yes, we do need help as they are a bit outdated compared to the current English version. 🙏 ! If you're ok with it, I would recommend with the React version first before moving on to the React Native version, as it's currently quite outdated, and I'm working with the React Native maintainer to get it updated. If you can make the adjustments soon, before the 7.0 release, I can reach out to the community and see if anyone is interested, or if you know someone that is willing to collaborate with you in proofreading it, I'd appreciate it.
> 
> Let me know and we'll go from there.
> 
> Hoping to see the pull request with the updates to the translation.
> 
> I hope you have a great day.
> 
> Stay safe

@jonniebigodes, Ok, well i'm glad to help :) 

Just to be clear about the scope, do you prefer that i translate in french one specific guide that has not been translated yet (or all not translated?), or also that i check all the english versions, and try to adapt the existing guides that have been translated (for instance: [learnstorybook.com/content/intro-to-storybook/react/](https://github.com/chromaui/learnstorybook.com/tree/master/content/intro-to-storybook/react)?)

**Comment by lmac-1** [comment]
If it helps at all, what I just did for Spanish was have a look at the current English React Intro To Storybook tutorial and then update the Spanish version file by file to make sure it matched the current English version. You're mostly rewriting the files, but I think I had to add one extra file that wasn't in the old Spanish version. 

You can find the English markdown files for the React Intro to Storybook tutorial [here](https://github.com/chromaui/learnstorybook.com/tree/master/content/intro-to-storybook/react/en)

**Comment by jonniebigodes** [comment]
@xaviermarchal, exactly what @lmac-1 mentioned. That's the best approach to this. If memory serves me right, the last update to it was back in 6.4.

**Comment by xaviermarchal** [comment]
@jonniebigodes ok, seems clear to me, i'll start with all files from `content/intro-to-storybook/react`, compare the french files with the english ones on the master branch, and push modifications on my branch if needed :)

I started with `composite-component.md` and correct me if i'm wrong, but i don't see an standard for branch/commit formatting, is there one? Also when i push on the branch i created in local, i have `permission denied`. Is there something i have to do?

**Comment by meanwhile7** [comment]
I can translate the docs in various languages with really good accuracy. Would love to update the outdated translation if available and add new language support to docs.

**Comment by ludovicobesana** [comment]
Hi @domyen :)
I can help with the Italian translations 🇮🇹

**Comment by jonniebigodes** [comment]
Hi @ludovicobesana, that's great to hear, and we would appreciate an Italian translation. 

Let me know of any questions you may have regarding them, and I'd be more than glad to follow up with you on them. Looking forward to seeing the pull request up.

Hope you have a fantastic day.

Stay safe

**Comment by tapi0623** [comment]
こんにちは
私はReactを使用している観点からStorybookを日本語に翻訳できます。

ぜひ、お手伝いさせてください。

**Comment by jonniebigodes** [comment]
Hey @tapi0623, if you're ok with it, can you start with the Intro to Storybook tutorial and also to unblock you and us for review? Can you push the entire updated tutorial as a single pull request? 

Looking forward to hearing from you.

Hope you have a fantastic week.

Stay safe

**Comment by tapi0623** [comment]
@jonniebigodes 

Thank you for getting back to me.
I'll accept that proposal and get right to work. 

Best wishes

**Comment by Halasalhab** [comment]
Hi @eihabkhan, @merodiro, I can help with React Arabic.

**Comment by lijo-belardi** [comment]
Hi @domyen,  I can help with the Italian translation. 😊

**Comment by jonniebigodes** [comment]
@lijo-belardi, thanks for your interest in helping us out with the translations. We already have one up, thanks to @ludovicobesana and @giowhatever. But we'd welcome additional ones. Let us know once the pull request is up so we can follow up with you and get it proofread. Sounds good? 

Have a fantastic day.

Stay safe

**Comment by nagashimam** [comment]
@jonniebigodes 
[I updated Japanese translation for Angular version of introduction to Storybook](https://github.com/chromaui/learnstorybook.com/pull/746). Could you find someone who can peer-review this PR?

**Comment by jonniebigodes** [comment]
@nagashimam I saw it and already followed up on the pull request. Cannot thank you enough for the continued help in getting the translations updated. We truly thankful for the amazing work you've been doing 🙇

**Comment by UHAsikakutou** [comment]
Hi @jonniebigodes , I can help with the Japanese translation of React parts.
However, it appears that the "Intro to Storybook" section you mention [here](https://github.com/chromaui/learnstorybook.com/issues/3#issuecomment-1734770403) has already been translated to the latest information.
Which section should I edit next?

**Comment by jonniebigodes** [comment]
@UHAsikakutou appreciate your interest in getting the translation updated. To give you some context, the Japanese version of the Intro to Storybook tutorial is quite outdated. As you can see [here](https://github.com/chromaui/learnstorybook.com/blob/master/gatsby-config.js#L27), it's still on Storybook version 6.5, which was released quite some time ago and also contains information that is quite outdated. We'd welcome your willingness to help us update the translation. I'd start by looking at the English version and begin collecting the changes required when you're able. Submit a pull request and ping me so that I can take a look from a technical perspective, and we can reach out to the community for a grammar/syntax review. Does that sound reasonable to you? Let me know, and we'll go from there.

Have a great day.

**Comment by wendy-YW** [comment]
Hey @jonniebigodes by any chance, do you still need help with Simplified Chinese translations? I can help with it for the React Native part if you want.

**Comment by justine-tarlet** [comment]
Hello! Do you need any help with French translations? I see that there is no French version for **Design Systems for Developers** , **UI Testing Handbook** and **Visual Testing Handbook**, so if needed, I can translate one of them 😃

**Comment by yziori** [comment]
Hello, @jonniebigodes 

I am interested in supporting the translation of the React section into Japanese. Please let me know how I can contribute to this project.

Thank you.

**Comment by lmac-1** [comment]
Hello! I recently [raised a PR](https://github.com/chromaui/learnstorybook.com/pull/775) to update the Intro to Storybook translation for Spanish (React).

I noticed that there's no Spanish translation for Design Systems for Developers (React), so I'm going to start working on that. Let me know if there's any other Spanish translations that should be prioritised instead. Also, if anyone would like to collaborate on the Spanish translations, let me know :)

**Comment by jonniebigodes** [comment]
Hi @lmac-1 👋 ! Thank you once again for helping us improve our translations. We appreciate it 🙏 ! If you're willing to, we'd appreciate the Design system for developers' tutorial translation to Spanish and with that, hopefully help the Spanish frontend community. Let me know if you have any questions so that I'm able to help you as much as I can.

Looking forward to seeing the pull request up.


Have a great day.

**Comment by pss-aileen** [comment]
Hello. I'd like to contribute to the Japanese translation.

- I found: 
  - PR: #765 
  - Comment: https://github.com/chromaui/learnstorybook.com/issues/3#issuecomment-2110616162

Is this PR almost done?
Is there anything I can help with?

**Comment by justine-tarlet** [comment]
Hello everyone! 
Here is the PR for the **french translation of design systems for developers (react)** : https://github.com/chromaui/learnstorybook.com/pull/777 
Feel free to review it! This is my first contribution so I hope I opened the PR and translated the right way. Don't hesitate if there is any problem with it :pray: 
If needed , I can update the french translation of **Intro to Storybook** to keep it up-to-date with english version. If other french contributors want to work on translations/updates, let me know!
Have a nice day

**Comment by wihyanghoon** [comment]
Hello, I’m interested in contributing a Korean translation for the Vue version of “Learn Storybook”. I’d like to confirm if anyone is already working on it before I start. If not, I’d be happy to begin.

Thanks!

**Comment by jonniebigodes** [comment]
@wihyanghoon, I appreciate you reaching out and wanting to get the Vue version translated into Korean 🙏 ! That's amazing to hear. You're more than good to go and get the translation in. Also, so you know, I'm working on getting the Vue edition updated to the latest Storybook version, and I'll be more than glad to follow up with you once it's in and get the translation updated as well.
Hope you have a fantastic weekend.

Stay safe

**Comment by 2nofa11** [comment]
Thank you for the Intro to Storybook tutorial — it has been really helpful for me.

I’ve opened a pull request that adds a Japanese translation for the Vue edition of the ‘Intro to Storybook’ tutorial based on the latest English version: #829.

If there’s anything that should be adjusted in terms of wording, structure, or locale configuration, I’m happy to update the PR.
